tiledb/sm/c_api/tiledb.h

```diff
@@ -422,7 +422,7 @@ tiledb_vfs_mode_from_str(const char* str, tiledb_vfs_mode_t* vfs_mode);
  * each individual dimension with the `set_buffer` API. Consult the current
  * documentation for more information.
  */
-TILEDB_DEPRECATED_EXPORT const char* tiledb_coords();
```

Comment:
This is deprecated, don't we want the export marked as such for the deprecation warnings?

Reply:
Ideally, but not at the cost of compile-time warnings when it not explicitly used by the user. The problem is the cpp-api that uses this. One viable alternative would be to hard-code the "__coords_" string in the cpp-api where it uses the `TILEDB_COORDS` macro. What do you think?